go/kbnm/hostmanifest/whitelist.go

```diff
@@ -2,10 +2,21 @@ package hostmanifest
 
 import (
 	"encoding/json"
+	"fmt"
 	"os"
 	"path/filepath"
 )
 
+func wrapWriteErr(err error, hostsPath string) error {
+	if !os.IsPermission(err) {
+		return err
+	}
+	dirName := filepath.Dir(hostsPath)
+	return fmt.Errorf("%s: Make sure you are the owner of the directory. "+
+		"You can run:\n "+
+		"  sudo chown -R $(whoami):staff %q", err, dirName)
```

Comment:
Please do a quick spotcheck on which splits it up: https://github.com/keybase/client/pull/7571/commits/015f15b15221005d314cc4af166a37885a2cbec8